lib/collection/src/operations/universal_query/collection_query.rs

```diff
@@ -56,6 +58,23 @@ pub struct CollectionQueryResolveRequest<'a> {
     pub using: String,
 }
 
+/// Internal representation of a group query request, used to converge from REST and gRPC.
+#[derive(Debug)]
+pub struct CollectionQueryGroupsRequest {
+    pub prefetch: Vec<CollectionPrefetch>,
+    pub query: Option<Query>,
+    pub using: Option<String>,
+    pub filter: Option<Filter>,
+    pub params: Option<SearchParams>,
+    pub score_threshold: Option<ScoreType>,
+    pub with_vector: Option<WithVector>,
+    pub with_payload: Option<WithPayloadInterface>,
+    pub group_by: JsonPath,
+    pub group_size: Option<usize>,
+    pub limit: Option<usize>,
+    pub with_lookup: Option<WithLookup>,
+}
```

Comment:
Just to be consistent with `CollectionQueryRequest`, I think we can resolve these default values when converting to this type:
- using
- with_vector
- with_payload
- group_size
- limit